WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -351,8 +384,10 @@ class PagesViewModel
     }
 
     private fun publishPageNow(remoteId: Long) {
-        pageMap[remoteId]?.date = Date()
-        changePageStatus(remoteId, PageStatus.PUBLISHED)
+        performIfNetworkAvailable {
+            pageMap[remoteId]?.date = Date()
+            changePageStatus(remoteId, PageStatus.PUBLISHED)
+        }
```

Comment:
what id there's no network available but only right at this moment, should we return something, or tell something to the user?

Reply:
`performIfNetworkAvailable` is a function. It [prints a message](https://github.com/wordpress-mobile/WordPress-Android/pull/8936/files/909ed4e14ad3188f7eeea9463f037f82288868da#diff-2b7e7932d5393c1ad246d3584ccf8cecR359) if the network's down.

Reply:
Yes saw that later so I resolved to continue and merge, then forgot to make the clarification here. Thanks!